### PR TITLE
Frontend Changes

### PR DESCRIPTION
**Pull request by KiaanCastillo**
# Changes
- Added red indicator line to slider for the start of prediction dates
- Added commas to stats values
- Modified slider to include new water prediction dates
- Modified padding values for the container class
## Screenshots
image

### CODE REVIEW

**Review by Alex0Blackwell — APPROVED** [review]
These updates look great!

File: App/js/UserInterface.js [hunk 1]
```diff
@@ -240,11 +240,11 @@ class UserInterface {
...
-    document.getElementById('min-consume').innerHTML = this.api.getMinConsumption();
-    document.getElementById('avg-price').innerHTML = '$'+this.api.getAvgPrice();
-    document.getElementById('max-price').innerHTML = '$'+this.api.getMaxPrice();
-    document.getElementById('min-price').innerHTML = '$'+this.api.getMinPrice();
+    document.getElementById('avg-consume').innerHTML = `${this.api.getAvgConsumption().toLocaleString('en')} HCF`;
+    document.getElementById('max-consume').innerHTML = `${this.api.getMaxConsumption().toLocaleString('en')} HCF`;
+    document.getElementById('min-consume').innerHTML = `${this.api.getMinConsumption().toLocaleString('en')} HCF`;
+    document.getElementById('avg-price').innerHTML = `$${this.api.getAvgPrice().toLocaleString('en')} USD`;
+    document.getElementById('max-price').innerHTML = `$${this.api.getMaxPrice().toLocaleString('en')} USD`;
+    document.getElementById('min-price').innerHTML = `$${this.api.getMinPrice().toLocaleString('en')} USD`;
```
Comment (Alex0Blackwell) [inline App/js/UserInterface.js]: I think we were also planning on rounding these values

Comment (KiaanCastillo) [reply App/js/UserInterface.js]: Rounding as in $795K instead of $795,599?

**Review by K-zhao — APPROVED** [review]

File: App/js/UserInterface.js (hunk 1 above)
Comment (Alex0Blackwell) [reply App/js/UserInterface.js]: No one said exactly but either $795,000 or $795K would be good!

**Comment by KiaanCastillo** [comment]
# Changes
- Modified stats to be displayed with nearest K or M
## Screenshot
image

**Review by Alex0Blackwell — APPROVED** [review]
Looks great!

**Review by jasonhuang16 — APPROVED** [review]

**Review by K-zhao — APPROVED** [review]
Looks great!

**Pull request merged**